### DISCUSSION TITLE
Quickdraw subtitle misspelling: 'gitty up' instead of 'giddy up'?

### DISCUSSION THREAD

**Discussion by dylandbl** [community/community#20554] [Other Feature Feedback, Questions, & Ideas] [answered]
Is this intentional? I've never seen 'giddy up' spelled this way. There are also significantly fewer results on Google for 'gitty up' than for 'giddy up':
![Screen Shot 2022-07-14 at 1 31 43 PM](https://user-images.githubusercontent.com/48745406/179068139-e78f0c46-c426-47ec-bec0-00c93b231dd9.png)
![Screen Shot 2022-07-14 at 1 31 35 PM](https://user-images.githubusercontent.com/48745406/179068150-0f28125b-fbb5-44b4-886e-dcf91184509d.png)

Can be seen in this example:
![Screen Shot 2022-07-14 at 1 37 10 PM](https://user-images.githubusercontent.com/48745406/179068918-1bfb4226-c1f4-483f-b456-c2e3d14a4087.png)

**Comment by radar-blip** [comment, marked as answer]
Im sure it was intentional. Git = Gitty

**Reply by dylandbl** [reply]
Ah, that makes sense.

**Reply by zeroidentidad** [reply]
xd

**Reply by react-dev-uz** [reply]
Yes, now I understand why it's like this :)😄

**Comment by Itz-Hyperz** [comment]
It's a play on the word because it's `Git`hub, not `Gid`hub...

**Comment by mathcodes** [comment]
Aaaah. Now I git it.

**Comment by LeeSeungjae00** [comment]
Aaaah. Now I git it. too

**Comment by Grajales-K** [comment]
what does means this achievement.?

**Reply by YasinDehfuli** [reply]
Its because you Closed an issue or a pull request within 5 min of opening
Here is some more info:
https://github.com/Schweinepriester/github-profile-achievements#achievements

**Comment by mtafadar** [comment]
Now I know
